src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/_help.py

```diff
@@ -105,6 +105,43 @@
     short-summary: Deletes a firewall rule in a Data Lake Store account.
 """
 
+helps['dls account virtual-network'] = """
+    type: group
+    short-summary: (PREVIEW) Manage Data Lake Store account virtual network rules.
+"""
+
+helps['dls account virtual-network create'] = """
+    type: command
+    short-summary: Creates a virtual network rule in a Data Lake Store account.
+    parameters:
+        - name: --subnet-id
+          type: string
+          short-summary: 'The subnet id for the virtual network rule.'
+        - name: --virtual-network-rule-name
```

Comment:
This should be aliased to `--name/-n` and that implies there's a good chance that `account_name` (which is likely aliased to --name) needs to be aliased to `--account-name` for this command group.